docs/troubleshoot/app-framework/zss-error-codes.md

```diff
@@ -193,15 +193,29 @@ The following error message codes may appear on ZSS log. Use the following messa
 
 ### ZWES1601I
 
-  Server is ready to accept JWT `with`(or `without`) fallback to legacy tokens
+ZSS server is ready to accept JWT with (or without) fallback to legacy tokens.
 
-  **Reason:**
+**Reason:**
 
-  Server is ready to accept JWT `with` or `without` fallback to legacy tokens.
+This message indicates that the ZSS (Zowe Security Server) has successfully started and is ready to accept JWT (JSON Web Token)-based authentication. Depending on your configuration, it may also accept legacy Zowe tokens as a fallback.
```

Comment:
ZSS is not known as the "Zowe Security Server"
zowe.org and zss on github call it "Zowe System Services (ZSS)"